<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="griffon.exceptions.InstanceMethodInvocationExceptionSpec" tests="3" skipped="0" failures="0" errors="0" timestamp="2016-12-02T13:04:58" hostname="testing-gce-cc5ab124-7f74-4323-a4b5-4e8dc3b1534d" time="0.017">
  <properties/>
  <testcase name="Exception should have proper formatted message (1)[0]" classname="griffon.exceptions.InstanceMethodInvocationExceptionSpec" time="0.008"/>
  <testcase name="Exception should have proper formatted message (1)[1]" classname="griffon.exceptions.InstanceMethodInvocationExceptionSpec" time="0.0"/>
  <testcase name="Exception should have proper formatted message (2)[0]" classname="griffon.exceptions.InstanceMethodInvocationExceptionSpec" time="0.009"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
